CONTRIBUTING.md

```diff
@@ -0,0 +1,152 @@
+# Contributing
+
+## Installation for Local Development
+
+[Poetry][poetry-link] is used to manage the python development environment. 
+
+A simple way to create an environment with the desired version of python and poetry is to use [conda][conda-link].  E.g.:
+
+```bash
+conda create -n fgpyo -c conda-forge "python>=3.9" poetry
+conda activate fgpyo
+
+# --all-extras is required to install `mkdocs` and associated dependencies,
+# which are required for development 
+poetry install --all-extras
+```
+
+[rtd-link]:    http://fgpyo.readthedocs.org/en/stable
+[poetry-link]: https://github.com/python-poetry/poetry
+[conda-link]:  https://docs.conda.io/en/latest/miniconda.html
```

Comment:
Miniconda is often used for things like CI runners or Docker images. A typical user/developer will probably want to use Mamba:

```suggestion
[conda-link]:  https://github.com/mamba-org/mamba
```